lib/core/src/lib/comments/comment-list/comment-list.component.ts

```diff
@@ -47,7 +48,14 @@ export class CommentListComponent {
         this.clickRow.emit(comment);
     }
 
-    getUserImage(userId: string): string {
-        return this.commentsService.getUserImage(userId);
+    getUserImage(user: User): string | null {
+        if (!user) {
```

Comment:
This is a breaking change since `commentsService` has different implementations based on the context such as `NodeCommentsService`, `CommentProcessService` etc, if you need to modify the behaviour it should be done in corresponding `getUserImage` method